ui/lib/core/addon/components/certificate-card.hbs

```diff
@@ -20,13 +20,13 @@
       {{@data}}
     </code>
   </div>
-  <div class="is-flex-center has-background-white-bis has-side-padding-s has-top-bottom-margin-negative-m">
+  <div class="is-flex-center has-background-white-bis has-top-bottom-margin-negative-m">
     <Hds::Copy::Button
       @text="Copy"
       @isIconOnly={{true}}
       @textToCopy={{@data}}
-      class="transparent is-paddingless"
-      data-test-copy-button
+      class="transparent"
```

Comment:
what is this class doing? if it's removing the background and border from the copy button, make sure:
- the other background is differentiated for the different states (hover/active)
- there is a transparent border applied to the "button" emulation
- the focus is correctly applied (somewhere)

Reply:
it's to stylize the copy button the same as `@color="tertiary"` to match the other buttons here
![transparent](https://github.com/hashicorp/vault/assets/68122737/0162d1dc-178e-4bf9-aa61-f975b52ab171)